src/pages/signUp/detail/useDetail.tsx

```diff
@@ -42,7 +42,7 @@ export const useDetail = () => {
       jobTitle,
     };
     const res = await details(param);
-    dispatch(setUserProfile(res.data));
+    // dispatch(setUserProfile(res.data));
```

Comment:
why did you comment it? if we don't need please remove it

Reply:
Please let it remain. I wanna check again later